Packages/com.unity.inputsystem/InputSystem/Plugins/InputForUI/InputSystemProvider.cs

```diff
@@ -642,7 +642,7 @@ void UnregisterActions(Configuration cfg)
             // The Next/Previous action is not part of the input actions asset
             UnregisterNextPreviousAction();
 
-            UnityEngine.Object.Destroy(m_InputActionAsset); // TODO check if this is ok
+            UnityEngine.Object.DestroyImmediate(m_InputActionAsset);
```

Comment:
Same qeustion/comment as when @Pauliusd01 looked into this, shouldn't this be conditioned with #UNITY_EDITOR and use Destroy in player build?
https://docs.unity3d.com/ScriptReference/Object.DestroyImmediate.html

Reply:
I've made the change @ekcoh suggested.
Although, as the information given to @Pauliusd01 states, there really isn't a good solution here except for making changes to how the editor works (which of course we cannot do).
Testing doesn't reveal any issues with 'DestroyImmediate' (or with Hakans suggestion).